### PR TITLE
Activate Account

### PR DESCRIPTION
**Pull request by emmytran**
# What does this PR solve and how
## What is this PR about?
- Front-end for User Signup

## How to run it step by step
# Run
`` npm start ``

# Implementation 
- New ActivateAccount.js file
- Added exact path (line 18 in App.js)
``  <Route exact path='/activate/:uid/:token' element={<ActivateAccount />} /> ``
- Added activateAccount function (line 42-50 in Auth.server.js)
`` 
const activateAccount = (uid, token) => {
    const config = {
        headers: {
            'Content-Type': 'application/json'
        }
    }
    const body = JSON.stringify({uid, token});
    return axios.post(`${API_URL_USERS}/activation/`, body, config);
}
``

- Added activateAccount under authService (line 57 in Auth.Server.js)

# Risk
- 

# Dependencies
- 

# PR Checklist
- [ ] Documentation
- [ ] Unit testing passed
- [ ] Code review and discussion
- [ ] Approve
- [ ] Merge

### CODE REVIEW

**Comment by longnguyen-isods** [comment]
Screenshot from 2023-04-23 14-45-42

1. Activate account page shouldn't have UserID or Token or anything to display to User. Either have a button, when user click that button, it will send a post request to Backend and display a message something like: Thank you for activation your account, now you can log-in your account... (Creativity is encouraged).

2. Your code couldn't send a post request to Backend, it showed "Activation failed" in 1 second and nothing happened. Which Backend didn't receive anything. ** I found your bug but I want you to find it.

**Review by longnguyen-isods — CHANGES_REQUESTED** [review]
Screenshot from 2023-04-23 14-45-42

1. Activate account page shouldn't have UserID or Token or anything to display to User. Either have a button, when user click that button, it will send a post request to Backend and display a message something like: Thank you for activation your account, now you can log-in your account... (Creativity is encouraged).

2. Your code couldn't send a post request to Backend, it showed "Activation failed" in 1 second and nothing happened. Which Backend didn't receive anything. ** I found your bug but I want you to find it.

3. Also, I just updated dev branch, so please update your local then fix your bug and push again Emmy. 

Thank you!

File: src/Components/services/Auth.server.js
Comment (longnguyen-isods) [inline src/Components/services/Auth.server.js:1]: image

1. Axios is already imported at line 1. By importing another axios at line 66, your code doesn't work. 
2. After I clicked the link, it sent request to Backend (GREAT!!). But it won't navigate through any other pages? --> I want the feature will return a message and navigate back to sign-in page. 
--> ActivateAccount.js ---- line 32 - 35 unreachable.

**Review by longnguyen-isods — CHANGES_REQUESTED** [review]

**Comment by emmytran** [comment]
Requested changes have been implemented. I added a function where it will automatically navigate to the sign-in page after the account is activated successfully (4 seconds)

**Pull request merged**